test/spmd/tensor/parallel/test_tp_examples.py

```diff
@@ -36,78 +34,6 @@ def forward(self, x):
         return self.net2(self.relu(self.net1(x)))
 
 
-def _aggregate_local_tensor(module: torch.nn.Module) -> torch.nn.Module:
```

Comment:
Can you also update test_2d_parallel.py? Not needed to be in the same PR. Thanks!